source/extract.py

```diff
@@ -0,0 +1,27 @@
+from utils import *
+
+config = load_config()
+
+class rbsa :
```

Comment:
There's another `class rbsa` in a file called `clean.py`, is this accidental? Their definition is identical, so I assume so...